docs/data-collection.md

```diff
@@ -84,6 +84,24 @@ For the OpenShift 4 Developer Preview we will be sending back these exact attrib
   // subscription_sync_total is the number of times an OLM operator
   // Subscription has been synced, labelled by name and installed csv
   '{__name__="subscription_sync_total"}',
+  //
+  //OCS metrics to be collected:
+  // ceph_cluster_total_bytes gives the size of ceph cluster in bytes.
+  '{__name__="ceph_cluster_total_bytes"}',
+  // ceph_cluster_total_used_raw_bytes is the amount of ceph cluster storage used in bytes.
+  '{__name__="ceph_cluster_total_used_raw_bytes"}',
+  // ceph_health_status gives the ceph cluster health status
+  '{__name__="ceph_health_status"}',
```

Comment:
Is this just a 1/0 yes/no? Can this be inferred from the up metric instead of adding a new one?

Reply:
This has values 0, 1 & 2. This 'ceph_health_status' metric presented is dependent on some internal ceph logic and thus cannot be inferred from 'up' metric.

This has cadinality 1.